src/receivers/ExpressReceiver.ts

```diff
@@ -258,7 +258,7 @@ export default class ExpressReceiver implements Receiver {
     this.app.use(this.router);
   }
 
-  private async requestHandler(req: Request, res: Response): Promise<void> {
+  public async requestHandler(req: Request, res: Response): Promise<void> {
```

Comment:
I hope moving this to `public` is OK, otherwise, testing this was extremely difficult - I would need to orchestrate both the Receiver and the App very specifically, and trigger entire middleware chain processing in order to test this private method.

Reply:
I'll let Kaz make the call on whether this is okay to move to public.

Reply:
Ideally, non-public is safer but I don't see any big risks by doing this. Go ahead 👍